package.json

```diff
@@ -20,7 +20,7 @@
         "lint": "eslint . --ext .js,.ts,.tsx && yarn workspace semaphore-contracts lint",
         "format": "prettier -c . && yarn workspace semaphore-docs format",
         "format:write": "prettier -w . && yarn workspace semaphore-docs format:write",
-        "docs": "typedoc --cname js.semaphore.pse.dev --githubPages true",
+        "docs": "typedoc",
```

Comment:
`cname` can be provided via config file.
And `githubPages` default to true
see
https://typedoc.org/schema.json

Reply:
amazing, +1